src/DefaultRoute.ts

```diff
@@ -0,0 +1,38 @@
+import Promise from 'dojo-core/Promise';
+import PathRule from './PathRule';
+import RouteManager from './RouteManager';
+import { CancelableNavigationArgs, DefaultRouteArgs, MatchableRoute, NavigationArgs } from './routing';
+
+export default class DefaultRoute implements MatchableRoute {
+	protected _rule: PathRule;
+
+	beforeEnter: (kwArgs: CancelableNavigationArgs) => (Promise<void> | void);
```

Comment:
Maybe... (I have one reservation about that rename, see https://github.com/dojo/core/issues/28)